lms/djangoapps/verify_student/views.py

```diff
@@ -1032,31 +1032,12 @@ def _send_confirmation_email(self, user):
         Send an email confirming that the user submitted photos
         for initial verification.
         """
-        if use_new_templates_for_id_verification_emails():
-            lms_root_url = configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL)
-            new_template_context = {
-                'user': user,
-                'dashboard_link': '{}{}'.format(lms_root_url, reverse('dashboard'))
-            }
-            return send_verification_confirmation_email(new_template_context)
-
-        context = {
-            'full_name': user.profile.name,
-            'platform_name': configuration_helpers.get_value("PLATFORM_NAME", settings.PLATFORM_NAME)
+        lms_root_url = configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL)
+        new_template_context = {
```

Comment:
the variable name is confusing. Maybe something like `new_email_template_context` or simply `context` would do.

Reply:
+1